build.gradle

```diff
@@ -0,0 +1,199 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath('io.swagger:swagger-codegen:3.0.0-rc0')
+    }
+}
+
+plugins {
+    id 'java'
+    id "org.jetbrains.kotlin.jvm" version "1.2.30"
+    id 'eclipse'
+    id 'idea'
+    id 'jacoco'
+    id 'checkstyle'
+    id 'findbugs'
+}
+
+description = "Chronicle REST APIs"

```

Comment:
Are you sure `build.gradle` is correct? It seems like there's a lot of stuff in here that might not be necessary (like odata dependency).